### ISSUE TITLE
WebAPI update

### ISSUE THREAD

**Issue by MitchMilam** [XrmServiceToolkit/XrmServiceToolkit#4]
Any timeframe on adding WebAPI support?

**Comment by con-ssc** [comment]
Currently there is no ETA on that, internally we are discussing how to manage it for our company. I will add functionality in my free time afterwards. I'll update the page on progress.

**Comment by chunxiao1** [comment]
any update on the timeframe for the WebAPI support?

**Comment by con-ssc** [comment]
No update, doesn't seem to be a high demand currently. Also, since there are so many endpoint versions this would be kind of hard.

**Comment by derekfinlinson** [comment]
If anyone is interested, I developed a [Web API library](https://github.com/derekfinlinson/xrm-webapi) you can use with JavaScript or TypeScript.

**Comment by ghost** [comment]
@derekfinlinson Gonna have a look. So have you found a way to always find the correct api version without user input? :)

**Comment by derekfinlinson** [comment]
@con-ssc No. I just let the user pass in the correct version in the constructor.

**Comment by ghost** [comment]
Okay got it.

**Comment by nsteenbeek** [comment]
This [Web API library](https://github.com/dys-solutions/crm-sdk) finds the correct api version for you without user input.
